### DISCUSSION TITLE
Correcting Model Parameters with GRACE-Derived Groundwater storage

### DISCUSSION THREAD

**Discussion by Chengwenju** [iiasa/CWatM#98] [Q&A] [answered]
hello,  everyone!

I am planning to use the CWatM model to simulate groundwater storage in an arid region. While calibrating the model parameters I chose the groundwater storage data exported by GRACE as the observations, but I found that the model is unable to simulate the rapid downward trend of groundwater storage, no matter how I adjust the parameter values (This is the first problem).

Also, I tried to set an initial value of groundwater storage, for example, I set it to 2m, but the result is that the groundwater storage shows a rapid decline, 1.5m or even more within a year, which is obviously wrong, but I don't understand how this error occurs (This is the second problem).

I hope you can help me to solve this problem.
@mikhailsmilovic @antosubash @khaeru @jensdebruijn @dof1985 @deepak-shah-np 
![calibration_result](https://github.com/iiasa/CWatM/assets/97773815/56dd054e-959f-4a3c-a4f9-0895e586a0fb)
![groundwater_section](https://github.com/iiasa/CWatM/assets/97773815/ba0cf102-e298-4e95-8c5f-f42d26b865f6)

**Comment by mikhailsmilovic** [comment]
This [video](https://youtu.be/aWQa0ZdYaWY?si=7DXLm9NZeq3UI8ky) goes through preparing total water storage outputs to compare with GRACE. An update from the video, we now suggest generally for limitAbstraction = False to allow for fossil water abstractions. 

Specifically for this question:
- What region are you modelling?
- Have you performed a warm-up? This may resolve the drop seen at the beginning.
- To confirm, are you using the version of CWatM without Modflow?
- Is the option limitAbstraction = False
- In your figure, what variable are you using for the Y-axis? For this purpose, tws would be recommended. 
Otherwise, to focus on Groundwater, you could take storGroundwater minus unmetDemand (this is fossil water abstraction).
- What version of CWatM are you using -- we would recommend the most recent version available in the main branch.

**Reply by Chengwenju** [reply]
Thank you for your reply

1. The region we are modeling is northwestern China, an arid land with low precipitation and groundwater pumping occurring mainly in oasis areas.
2. I tried load_initial=True, but the difference between the simulated groundwater storage ("storGroundwater_monthavg" variable) and the GRACE exported groundwater storage is very large, as shown in Figure 1. No matter how the parameters are adjusted, the problem cannot be solved.
3. I am using the version of CWatM without Modflow.
4. I confirm the option limitAbstraction = False
5. We similarly found that the simulated change in groundwater storage ("storGroundwater_monthavg" variable from the CWatM model output) was significantly smaller than the GRACE-derived change in groundwater storage when we calibrated using TWS, and it is not clear to us what is causing this problem. Because we are primarily concerned with groundwater, we tried to set up a calibration in the "calibration_single.py" file to use the "storGroundwater_monthavg" variable for calibration against the GRACE-derived groundwater storage change, but the results were very unsatisfactory (KGE<0). Also, why subtract the unmet water demand from the groundwater storage instead of summing the two variables?
6. The version we use is CWatM-version 1.06, downloaded September 2022.

What I'm most interested in now is how I can make my simulation results match the GRACE derived groundwater storage changes, the parameters I'm calibrating include
SnowMeltCoef = %SnowMelt
crop_correct = %crop
soildepth_factor = %soildepthF
preferentialFlowConstant = %pref
arnoBeta_add = %arnoB
factor_interflow = %interF
recessionCoeff_factor = %reces
runoffConc_factor = %runoff
manningsN = %CCM
normalStorageLimit = %normalStorageLimit
lakeAFactor = %lakeAFactor
lakeEvaFactor = %lakeEvaFactor
swAbstractionFrac = %swAbstractionFrac
soil_alpha= %soil_alpha
matric_b= %matric_b

I tried a lot of methods and areas but I can't achieve my purpose, I don't know if there is an error in my settings in the "calibration_single.py" file, I sincerely hope you can help me to solve this problem, thank you very much!
(Attached are settings_monthly1.ini and calibration_single.py)

[gws_cali.zip](https://github.com/iiasa/CWatM/files/13280242/gws_cali.zip)

![figure 1](https://github.com/iiasa/CWatM/assets/97773815/f7c2b719-b6c7-417a-8aee-f04b25627e62)

**Reply by mikhailsmilovic** [reply, marked as answer]
Guten Tag,
- Would you provide the shapefile for your region of interest somewhere?
- We recommend using the most updated version of the model, available in the _main_ branch. 
- For comparisons with GRACE, we recommend the variable tws (total water storage). Otherwise, storGroundwater minus unmetDemand. It is subtracted as storGroundwater only considers "recent" groundwater, and unmetDemand is all fossil water abstraction. This is to observe changes in groundwater storage.
- GRACE typically compares changes in total water storage relative to the 2004-2009.99 period. In the calibration, it seems storage (and not changes in storage) is being looked at. In this case, it would be recommended to compare with CWatM total water storage relative to the CWatM total water storage from 2004 to 2009.99.

**Reply by Chengwenju** [reply]
Thank you for your reply

Attached is the shapefile we are interested in. We'll keep trying based on your comments

[Tarim.zip](https://github.com/iiasa/CWatM/files/13281451/Tarim.zip)

**Reply by hsk0059** [reply]
@Chengwenju 
 Dr. Cheng, may I have a discussion with you regarding Groundwater storage of CWatM-Modflow, I have a similar problem as yours.
